Firmware/UI/Screens/MainPage.cpp

```diff
@@ -60,21 +60,9 @@ void MainPage::DigitalGainButtonHandler(void* sender)
 
 void MainPage::Draw(IPainter* painter)
 {
-    /*painter->DrawIcon(apertus_text.pixel_data_text, 58, 89, apertus_text.width, apertus_text.height,
-                      RGB565(194, 191, 188));
-    painter->DrawIcon(apertus_ring.pixel_data_ring, 58 + apertus_text.width, 89, apertus_ring.width,
-                      apertus_ring.height, RGB565(244, 114, 72));
-                      */
-
-    painter->SetFont(Font::FreeSans9pt7b);
-    painter->DrawText(50, 80, "FreeSans9pt7b", (uint16_t)Color565::White, TextAlign::TEXT_ALIGN_LEFT, 0);
-    painter->SetFont(Font::FreeSans12pt7b);
-    painter->DrawText(50, 100, "FreeSans12pt7b", (uint16_t)Color565::White, TextAlign::TEXT_ALIGN_LEFT, 0);
-    painter->SetFont(Font::FreeSans18pt7b);
-    painter->DrawText(50, 130, "FreeSans18pt7b", (uint16_t)Color565::White, TextAlign::TEXT_ALIGN_LEFT, 0);
-    painter->SetFont(Font::FreeSans24pt7b);
-    painter->DrawText(50, 170, "FreeSans24pt7b", (uint16_t)Color565::White, TextAlign::TEXT_ALIGN_LEFT, 0);
-
+    painter->DrawIcon(ApertusTextLogo.Data, 58, 89, ApertusTextLogo.Width, ApertusTextLogo.Height, RGB565(194, 191, 188));
```

Comment:
Maybe we can just supply 'Icon*', as we draw them 1:1, so height and width won't change.

Reply:
What about the position?

Reply:
I mean, how'll we specify the x and y?

Reply:
How do we specify it at the moment?

Reply:
Like this -> https://github.com/MetalDent/AXIOM-Remote/blob/dev/Firmware/UI/Screens/MainPage.cpp#L63

Reply:
Ehm, i see the source code above and don't understand the problem to adjust the method to use Icon, without specifying data, width and height.

Reply:
And i know the source code, no need to point me to it. But i'm wondering where the difficulties at the moment are.

Reply:
Got it now!